src/transformState.js

```diff
@@ -5,7 +5,29 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
-}
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+
+      case 'removeProperties':
+        for (const toBeRemoved of action.keysToRemove) {
+          if (toBeRemoved in state) {
```

Comment:
I don't understand why don't you call it just `key`

Reply:
Sure , lets keep the names clear and simple, code dry  and be stick to the task.